### PR TITLE
fix(dropdown): update css of search tab

### PR DESCRIPTION
**Pull request by varnikajain15**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Comment by anuradha9712** [comment]
Please update the commit message

File: css/src/components/dropdown.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
 }
 
 .Dropdown-inputWrapper {
-  border-bottom: var(--border);
+  border-bottom: 1px solid #c5c5c5;
```
Comment (anuradha9712) [inline css/src/components/dropdown.css]: Please use CSS variable for particular color

Comment (varnikajain15) [reply css/src/components/dropdown.css]: Done

File: css/src/components/dropdown.css [hunk 2]
```diff
@@ -9,7 +9,7 @@
 }
 
 .Dropdown-inputWrapper {
-  border-bottom: var(--border);
+  border-bottom: var(--spacing-xs) solid var(--stone-dark);
```
Comment (anuradha9712) [inline css/src/components/dropdown.css]: Please use the CSS token available on this link https://mds.innovaccer.com/?path=/story/others-design-tokens-colors--colors

File: css/src/components/dropdown.css [hunk 3]
```diff
@@ -19,6 +19,7 @@
 .Dropdown-input,
 .Dropdown-input:focus-within {
   border: unset !important;
+  box-shadow: none !important;
 }
```
Comment (anuradha9712) [inline css/src/components/dropdown.css:23]: Please handle all the 3 states of search input "default", "focused", & "after search". Refer to this link https://www.figma.com/file/w8sqBtJpvq86D06UE7gN0T/MDS---Web?node-id=2748%3A30842

**Comment by anuradha9712** [comment]
Rebase your branch with `develop` branch

**Comment by anuradha9712** [comment]
Change the commit message

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**